content/authorization/application-registration-prerequisites.md

```diff
@@ -0,0 +1,207 @@
+# Application Registration Prerequisites / Developer Guide
```

Comment:
Shall we add a link under the main Authorization page to this page?

http://fhir.cerner.com/authorization/

```
[Registration](http://fhir.cerner.com/authorization/#registration)
    [Application Registration Prerequisites](...)
    [Registering a System Account](http://fhir.cerner.com/authorization/#registering-a-system-account)
```

Reply:
+1

Reply:
+1